api/app/settings.py

```diff
@@ -175,6 +179,9 @@
 CORS_ALLOWED_ORIGINS = [
     env('FRONTEND_URL')
 ]
+CSRF_TRUSTED_ORIGINS = [
+    env('FRONTEND_URL')
+]
```

Comment:
This one already exist in master branch, so please make sure they do not get duplicated, please do rebase if necessary

Reply:
@FranzDelosSantos yes